src/main/java/org/jenkinsci/plugins/GithubAuthorizationStrategy.java

```diff
@@ -140,6 +142,10 @@ public String getAdminUserNames() {
         return StringUtils.join(rootACL.getAdminUserNameList().iterator(), ", ");
     }
 
+    public String getAgentUserName() {
```

Comment:
Can you please add some javadoc to this method? You can follow other getters/setters as an example.